mesonbuild/interpreter/interpreter.py

```diff
@@ -877,7 +877,7 @@ def disabled_subproject(self, subp_name: str, disabled_feature: T.Optional[str]
     def do_subproject(self, subp_name: str, method: Literal['meson', 'cmake'], kwargs: kwtypes.DoSubproject) -> SubprojectHolder:
         disabled, required, feature = extract_required_kwarg(kwargs, self.subproject)
         if disabled:
-            mlog.log('Subproject', mlog.bold(subp_name), ':', 'skipped: feature', mlog.bold(feature), 'disabled')
+            mlog.debug('Subproject', mlog.bold(subp_name), ':', 'skipped: feature', mlog.bold(feature), 'disabled')
```

Comment:
Nope, we definitely want to log the fact that we are skipping the possibility of evaluating the subproject altogether.

Reply:
Ah, my bad. I got this confused with a similar log message used for dependencies (which I would like to be debug for subprojects):
```
            mlog.log('Dependency', mlog.bold(self._display_name), 'skipped: feature', mlog.bold(feature), 'disabled')
```

Reply:
changed this back to `log` in cb172a2fe099a27f4bd4f61940bd680b186eeb9d